addons/binding/org.openhab.binding.meterreader/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,43 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MeterReader Binding
+Bundle-SymbolicName: org.openhab.binding.meterreader;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/jsml.jar,
+ lib/j62056.jar,
+ lib/rxjava-2.2.0.jar,
+ lib/reactive-streams-1.0.2.jar
+Import-Package: 
+ com.google.common.base;version="15.0.0",
```

Comment:
Please remove the dependency on Guava (unless this is required by an included library).

Reply:
`com.google.common.base.Throwables` class is used by this binding

Reply:
@kaikreuzer: Is it ok to use this dependency?

Reply:
Should be avoided as we try to get rid off Guava and it seems that `Throwables` is indeed anything that couldn't be easily replaced.